src/Definition/Common.php

```diff
@@ -58,6 +72,22 @@ public function isNullable()
     /**
      * @return string
      */
+    public function getDefault()
```

Comment:
We do not need to support them in the API and we won't propagate the properties from the API call to the instances.